sweepbatcher/sweep_batch.go

```diff
@@ -998,6 +1004,36 @@ func (b *batch) musig2sign(ctx context.Context, inputIndex int, sweep sweep,
 		return nil, fmt.Errorf("invalid htlc script version")
 	}
 
+	var digest [32]byte
+	copy(digest[:], sigHash)
+
+	if b.cfg.signMuSig2 != nil {
+		// A custom signer is installed. Use it instead of
```

Comment:
nit: i'd suggest to move the comment above the if for readability.

Reply:
Fixed